src/integrationTest/java/uk/gov/hmcts/darts/testutils/stubs/DataManagementServiceStubImpl.java

```diff
@@ -39,7 +39,7 @@
 @Component
 @Slf4j
 @RequiredArgsConstructor
-@Profile("intTest")
+@Profile("intTest && !blobTest")
```

Comment:
We need a better way of excluding/including beans when test code is run. Production code should not know about test profiles. Maybe this can be address through a test specific configuration class in future.